exercises/defining_models/scaffold/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "author": "C J Silverio <ceejceej@gmail.com>",
+  "description": "the defining models exercise",
+  "keywords": [],
+  "license": "MIT",
+  "main": "index.js",
+  "name": "scaffold",
+  "version": "1.0.0",
+  "dependencies": {
+    "@hapi/iron": "^6.0.0",
+    "accepts": "^1.3.7",
+    "ajv": "^6.12.2",
+    "are-we-dev": "^1.0.0",
+    "bole": "^4.0.0",
+    "cookie": "^0.4.1",
+    "culture-ships": "^1.0.0",
+    "dotenv": "^8.2.0",
+    "find-my-way": "^3.0.4",
+    "ormnomnom": "^7.1.0",
+    "pg": "^8.3.3",
+    "uuid": "^8.3.0"
+  },
+  "devDependencies": {
+    "@hapi/shot": "^5.0.4",
+    "@typescript-eslint/eslint-plugin": "^4.11.0",
+    "@typescript-eslint/parser": "^4.11.0",
+    "ansi-escapes": "^4.3.1",
+    "bistre": "^1.0.1",
+    "eslint": "^7.1.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.3",
+    "get-function-location": "^2.0.0",
+    "nodemon": "^2.0.4",
+    "prettier": "^2.0.5",
+    "supports-hyperlinks": "^2.1.0",
+    "tap": "^14.10.7"
+  },
+  "scripts": {
+    "boltzmann:docs": "npx boltzmann-cli --docs",
+    "boltzmann:routes": "node -e 'require(\"./boltzmann\").printRoutes()'",
+    "boltzmann:upgrade": "npx boltzmann-cli",
+    "lint": "eslint .",
+    "posttest": "npm run lint",
+    "start": "nodemon ./boltzmann.js",
+    "test": "boltzshopper run ."
```

Comment:
I added `"sql": "boltzshopper run __example 2>&1; docker exec -it pg-1bb8346b-2312-4cd6-a1b3-0f2ebf6d2bba psql postgres://postgres:postgres@localhost/postgres",` as well, to let folks get a SQL shell to see what's going on. I might revisit and add this / drop in a `.env` with the PGURL as well